OMX-27-firmware/OMX-27-firmware.ino

```diff
@@ -819,36 +900,36 @@ void dispGenericMode(int submode, int selected){
 
 
 	switch(selected){
-		case 0:
+		case 1:
 			display.fillRect(0*32+2, 9, 29, 21, WHITE);
 			break;
-		case 1: 	//
+		case 2: 	//
 			display.fillRect(1*32+2, 9, 29, 21, WHITE);
 			break;
-		case 2: 	//
+		case 3: 	//
 			display.fillRect(2*32+2, 9, 29, 21, WHITE);
 			break;
-		case 3: 	//
+		case 4: 	//
 			display.fillRect(3*32+2, 9, 29, 21, WHITE);
 			break;
-		case 4: 	//
+		case 0: 	//
 			break;
 		default:
 			break;
 	}
 	// ValueBoxes
 	int highlight = false;
-	for (int j = 0; j<4; j++){
+	for (int j = 1; j<5; j++){
 
 		if (j == selected) {
```

Comment:
I think this is because had this previously where `selected = 4` would be no-highlight, but I changed it now where `0, 5, 10, etc` are no-highlight, so it is 0-indexed, but in that chunk I only want to highlight if its 1-4.